uiautomationutilities/perf_tool/readme.md

```diff
@@ -0,0 +1,155 @@
+# Perf tool walk through
+
+**Recording:** https://msit.microsoftstream.com/video/8930a1ff-0400-9887-b080-f1eb55ce8803
+
+**Source code of C# tool:** 
+- Repo: https://github.com/AzureAD/microsoft-authentication-library-common-for-android
+- Location: /uiautomationutilities/perf_tool
+
+**Pipeline information:** https://dev.azure.com/IdentityDivision/IDDP/_build?definitionId=1254
+
+### The source/path of the files:
+- **New files:**
+	- command.cs
+	- PerfDataConfiguration.xml
+	- PerfMeasurementConfigurationsProvider.cs
+	- Program.cs
+- **PerfCL files:**
+	- MathUtils.cs: 
+	- MeasurementsConfiguration.cs
+	- MeasurementsStore.cs
+	- PerfData.cs
+	- PerfDataConfiguration.cs
+	- PerfDataRecord.cs
+	- PerfMeasurement.cs
+	- PerfMeasurementsSet.cs
+	- PerformanceMetric.cs
+	- ReportHelper.cs
+	- XMLUtility.cs
+- **PerfDiffResultMailer files:**
+	- Gen3MeasurementsData.cs -> Renamed to -> MeasurementsData.cs
+	- Parameter.cs
+	- Task.cs
+	- View.cs
+
+### Source details of the files:
+
+- **New files:** 
+	- Source: None
+	- Contact: Neerav Agarwal (Neerav.Agarwal@microsoft.com)
+- **PerfCL files:** 
+	- Source: \\daddev\office\16.0\13328.10000\src\toolsrc\PerfCLTool
+	- Contact: Arpit Aggarwal (Aggarwal.Arpit@microsoft.com)
+- **PerfDiffResultMailer files:** 
+	- Source: \\officestore\scratch\gargupta\PerfDiffResultMailer\
+	- Contact: Garima Gupta (Garima.Gupta@microsoft.com)
+
+
+### Steps to modify the code to add a new marker and see it in the report:
```

Comment:
Let's plan to do a brownbag on this topic; I think this would be useful to demonstrate